test/local/routes/sms.js

```diff
@@ -280,6 +313,67 @@ describe('/sms', () => {
   })
 })
 
+describe('/sms with signinCodes disabled', () => {
```

Comment:
By `disabled`, do you mean the feature isn't enabled on the auth-server or the content-server doesn't send `signinCodes`? I don't see any on/off gating logic in this PR, the closest thing I can find is whether [signinCodes is a member of features](https://github.com/mozilla/fxa-auth-server/pull/1904/files#diff-c8e31dade53c85c9af99636f696d7266R89).